src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -869,7 +869,7 @@ public void respondWantDigestTwoSupportedQvalueNonZeroExternalBinaryContentHead(
     public void respondWantDigestNonSupportedExternalBinaryContent(final String uri)
         throws FileNotFoundException {
         final TestInfo info =
-            setupTest("3.9-J-2", "respondWantDigestNonSupportedExternalBinaryContent",
+            setupTest("3.9-J-1", "respondWantDigestNonSupportedExternalBinaryContent",
```

Comment:
Instead of `3.9-J-1`, should this instead be named `3.9.3`?
https://fedora.info/2018/06/25/spec/#redirect-and-proxy